catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -44,6 +44,10 @@
     }
 
     public void setAttributeMap(List<String> attributeMappings) {
+        if (attributeMappings.size() == 0 || (attributeMappings.size() == 1 && (attributeMappings.get(0).isEmpty()))) {
```

Comment:
If this is called by ConfigurationAdmin when something's changed in the Admin Console, we should check for `null` as well.